opendevin/runtime/docker/ssh_box.py

```diff
@@ -477,7 +511,10 @@ def close(self):
         containers = self.docker_client.containers.list(all=True)
         for container in containers:
             try:
-                if container.name.startswith(self.container_name_prefix):
+                if container.name.startswith(self.container_name):
```

Comment:
Why we use the `container_name_prefix` in the previous time?

Reply:
I have no idea :(